crowdin.yml

```diff
@@ -2,5 +2,5 @@ files:
   - source: /udata_front/theme/gouvfr/translations/gouvfr.pot
     translation: >-
       /udata_front/theme/gouvfr/translations/%two_letters_code%/LC_MESSAGES/%file_name%.po
-  - source: /theme/js/locales/en.json
```

Comment:
We'll need to update vue-i18n-extract.config.js as well

Reply:
Because `vue-i18n-extract` wants a glob and we must also match `udata-front-plugins-helper`, I had to slightly change the match so it's targeting any `.js` or `.vue` file in a `js` folder in any folder in `udata_front` and not in `udata_front/theme`. I can change the `js`folder check to `assets` too but I don't know if it's better.

Reply:
Seems fine!